utils/systematics.py

```diff
@@ -0,0 +1,31 @@
+import awkward as ak
+import numpy as np
+
+def rand_gauss(item):
```

Comment:
If this corresponds to the implementation in `coffea`, is there a reason we cannot import and use that one directly?

Reply:
It's not exactly the same, in particular I had to slightly change the way the seed is computed (see [here](https://github.com/maxgalli/analysis-grand-challenge/blob/for_testing/analyses/cms-open-data-ttbar/for_testing.py)). The reason is that in my case it was failing due to a problem with array substructure, I believe because in my case there are events with no jets while in the case of the coffea function it is not intended to be applied on such samples (but I might be wrong).